package.json

```diff
@@ -66,60 +66,60 @@
     "report-dir": "coverage-cypress"
   },
   "devDependencies": {
-    "@cypress/code-coverage": "^3.8.7",
-    "@cypress/webpack-preprocessor": "^5.5.0",
+    "@cypress/code-coverage": "^3.9.4",
+    "@cypress/webpack-preprocessor": "^5.7.0",
     "@istanbuljs/nyc-config-typescript": "^1.0.1",
     "@svgr/webpack": "^5.5.0",
     "@testing-library/react-hooks": "^3.7.0",
     "@types/enzyme": "^3.10.8",
     "@types/enzyme-adapter-react-16": "^1.0.6",
-    "@types/jest": "^26.0.19",
-    "@types/react": "^16.14.2",
-    "@types/react-dom": "^16.9.10",
-    "@typescript-eslint/eslint-plugin": "^4.9.1",
-    "@typescript-eslint/parser": "^4.9.1",
-    "autoprefixer": "^10.1.0",
+    "@types/jest": "^26.0.22",
+    "@types/react": "^16.14.5",
+    "@types/react-dom": "^16.9.12",
```

Comment:
Given that we have React 17 in `dependencies`, we should really be using the React 17 version of the `@types/react*`, but that's a preexisting condition so feel free to kick that can down the road if you're so inclined.

Reply:
Good point. Done.